test/unit/memoCtrl.test.js

```diff
@@ -93,18 +71,14 @@ describe('Contol functions for combining data', () => {
   })
 
   test('Kopps data cleaned up from templates', done => {
-    const newKoppsData = memoCtrl.removeTemplatesFromKoppsFreshData(mockedKoppsTemplates)
-    expect(newKoppsData.equipmentTemplate).toBe(undefined)
+    const newKoppsData = memoCtrl.removeTemplatesFromKoppsFreshData(JSON.parse(JSON.stringify(mockedKoppsTemplates)))
     expect(newKoppsData.literatureTemplate).toBe(undefined)
-    expect(newKoppsData.possibilityToCompletionTemplate).toBe(undefined)
-    expect(newKoppsData.possibilityToAdditionTemplate).toBe(undefined)
-
     done()
   })
 
   test('Merge kopps data and api data, memo api data replaces kopps data', async () => {
     const newKoppsData = await memoCtrl.mergeKoppsCourseAndMemoData(
-      mockedKoppsTemplates,
+      JSON.parse(JSON.stringify(mockedKoppsTemplates)),
```

Comment:
If the above suggestions are applied, this parsing wont be needed.

```suggestion
mockedKoppsTemplates,
```